### ISSUE TITLE
Causes jump to the begining of file 

### ISSUE THREAD

**Issue by jank84** [saschanaz/AtomScroller#1]
Happens e.g. if you search via "Project Find: Show" something and click inside the editor.

**Comment by saschanaz** [comment]
Yep, I know this bug but currently don't know how to fix this. Thank you for posting this issue.
